genfragments/ThirteenTeV/HToZATo2L2B/Hadronizer_TuneCP5_13TeV_HToZATo2L2B_ggH_pythia8_PSweights_cff.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring(''),
+    nEvents = cms.untracked.uint32(5000),
+    numberOfParameters = cms.uint32(1),
+    outputFile = cms.string('cmsgrid_final.lhe'),
+    scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+)
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        parameterSets = cms.vstring('pythia8CommonSettings',
+                                    'pythia8CP5Settings',
+                                    'pythia8PSweightsSettings',
+                                    )
+    )
+)
+ProductionFilterSequence = cms.Sequence(generator)
```

Comment:
How is this different from normal CP5 tune? if nothing, please remove this commit

Reply:
No difference indeed, removed .